ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -32,6 +32,16 @@ def _ceil(
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
 
+#round
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def round(
+    x,
```

Comment:
All the tests are failing, add a "/" after `x` and before `out`, it is considering x as a keyword arg which is failing the test, this will fix it for `numpy`, it will still fail for other frameworks as there is an issue with the datatype you are generating, kindly look into it and ask for a review when the tests are passing or you hot a wall on a specific problem
Thanks!

Reply:
I will thank you